### DISCUSSION TITLE
console.log not working

### DISCUSSION THREAD

**Discussion by Shiv2116** [Cyfrin/foundry-full-course-cu#88] [Q&A] [answered]
shivdixit@Ubuntu:~/solidity-course/foundry-fund-me$ forge script script/Interactions.s.sol
[⠘] Compiling...
[⠒] Compiling 2 files with 0.8.20
[⠢] Solc 0.8.20 finished in 185.04ms
Error: 
Compiler run failed:
Error (7576): Undeclared identifier.
  --> script/Interactions.s.sol:14:9:
   |
14 |         console.log("Funded FundMe with ", SEND_VALUE);
   |         ^^^^^^^

shivdixit@Ubuntu:~/solidity-course/foundry-fund-me$

**Comment by alymurtazamemon** [comment, marked as answer]
@Shiv2116 Do you have this import `import "forge-std/console.sol";` inside interactons.s.sol
